pkg/envoy/proxy.go

```diff
@@ -95,11 +95,6 @@ func (p *Proxy) SetNewNonce(typeURI TypeURI) string {
 	return p.lastNonce[typeURI]
 }
 
-// String returns the CommonName of the proxy.
```

Comment:
In the past it seemed like the smart thing to do - to implement the Stringer interface.  But that's a trap - one could expose stuff accidentally in logs.  So decided to remove this.